### ISSUE TITLE
Unexpected token / in JSON at position 691

### ISSUE THREAD

**Issue by alan-rpi** [AtomLinter/linter-eslint#824]
Hi. I posted the problem below against the language-javascript module but they replied as follows:
---------------------
It appears that this is being caused by the [linter-eslint](https://atom.io/packages/linter-eslint) package, not Atom itself. It may be that there is still a bug in Atom. If so, we'll need the maintainer of the package that is causing the problem to help track that down. You should file this issue on their repository so they can either fix the problem or tell us where the problem is coming from. 
---------------------

Description
[Description of the issue]

When language-javascript is enabled get the following error with some js files

Error: H:\Home Data\Projects\House Monitoring System\aggregator by topic\package.json: Unexpected token / in JSON at position 691
SyntaxError: H:\Home Data\Projects\House Monitoring System\aggregator by topic\package.json: Unexpected token / in JSON at position 691
at Object.parse (native)
at Object.Module._extensions..json (module.js:573:27)
at Module.load (module.js:473:32)
at tryModuleLoad (module.js:432:12)
at Function.Module._load (module.js:424:3)
at Module.require (module.js:483:17)
at require (internal/module.js:20:19)
at Object.getConfigPath (F:\Atom.atom\packages\linter-eslint\lib\worker-helpers.js:138:30)
at global.onRequest (F:\Atom.atom\packages\linter-eslint\lib\worker.js:69:30)
at Emitter.emit (F:\Atom.atom\packages\linter-eslint\node_modules\sb-event-kit\lib\emitter.js:70:19)

Do not get when language-javascript is disabled or on another .js file
Steps to Reproduce

    [First Step]
    Enable language-javascript
    [Second Step]
    Create a new file
    Save as any name.js
    make a keystroke including ctrl-s to save
    [and so on...]
    after dismissing the error pane then any key strokes creates the error again.

This happens with existing .js files and new ones. See Additional Info below

Expected behavior: [What you expect to happen]
No error!

Actual behavior: [What actually happens]
Error message as given above

Reproduces how often: [What percentage of the time does it reproduce?]
Every time I focus on a .js file and then every time I make a keystroke.

NOTE; I spent many hours yesterday editing another js file without problems and that file today is OK. But I opened another file from 2015 and that immediately gives the same error. This suggests it's the contents of the file that is causing the problem.
Versions

Microsoft Windows [Version 6.1.7601] - Windows 7 Professional
Atom : 1.14.1
Electron: 1.3.13
Chrome : 52.0.2743.82
Node : 6.5.0
apm 1.15.3
npm 3.10.5
node 4.4.5 ia32
python
git
visual studio

You can get this information from copy and pasting the output of atom --version and apm --version from the command line. Also, please include the OS and what version of the OS you're running.
Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

I went to the original file in GitHub
https://github.com/jaggr2/node-red-contrib-aggregator/blob/master/aggregator/aggregator.js
copied the source and pasted into a new file in NotePad++ and saved. Then opened in Atom and immediately get the error.

**Comment by IanVS** [comment]
It looks to me like there may be an invalid character `/` in your `package.json`, `H:\Home Data\Projects\House Monitoring System\aggregator by topic\package.json`

**Comment by alan-rpi** [comment]
Ian, thanks for a quick reply.

That there should be a problem with the package..json file surprised me as I'm trying to edit the js. file but I recall the Atom can upload to github - I'm new to Atom and github.

I did scan the package.json file and could not see any obvious error but nevertheless I took a copy of another package.json and then edited it with the changes from my problem package.json. Now Atom works! 

Many thanks. 
Alan
